src/test/resources/internationalization/i18NCases_es.properties

```diff
@@ -21,4 +21,4 @@ popup.message.regexp=Se creó Caso "[0-9]{8}".
 title.case=Caso
 header.caseNumber=starts-with(@title,'N') and contains(@title,'mero del caso')
```

Comment:
Maybe it would be better to use only a contains, in the locator and avoid the starts with or try to prepare the string first?

Reply:
Yes, I'll first attempt to use the accents from a different file and check if it supports it. Otherwise I'll use the workaround you propose. This for the next pull request considering it is involved in several fields and would demand some time to refactor.